example4 Kopie/shapes/areaCalculator.py

```
@@ -0,0 +1,92 @@
+
+class Shape(ABC):
+    @abstractmethod
+    def area(self):
+        pass
+
+    @abstractmethod
+    def volume(self):
+        pass
+
+class AreaCalculator:
+    def __init__(self, shapes=[]) -> None:
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)